### ISSUE TITLE
Sesamoiditis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11187]
**Preferred term label:**
Sesamoiditis

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0264076
MRI of the foot revealing edema within one or both of the hallux sesamoids

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sesamoid injuries of the hallux

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001348
- label: Hallux sesamoiditis
- parents: [Abnormal hallux morphology (HP:0001844)](https://hpo.jax.org/app/browse/term/HP:0001844)
- pmid: PMID:32583133
- def: Inflammation of the sesamoid bones of the hallux ( two ovoid-shaped ossicles embedded within the tendons under the big toe). This feature can be ascertained by MRI of the foot, revealing edema within one or both of the hallux sesamoids.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11187
- synonyms:
